runtime/glbl.c

```diff
@@ -1337,31 +1404,31 @@ glblDoneLoadCnf(void)
 			}
 			free(proto);
 		} else if(!strcmp(paramblk.descr[i].name, "senders.reportnew")) {
-		        glblReportNewSenders = (int) cnfparamvals[i].val.d.n;
+	        glblReportNewSenders = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "senders.reportgoneaway")) {
-		        glblReportGoneAwaySenders = (int) cnfparamvals[i].val.d.n;
+	        glblReportGoneAwaySenders = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "senders.timeoutafter")) {
-		        glblSenderStatsTimeout = (int) cnfparamvals[i].val.d.n;
+	        glblSenderStatsTimeout = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "senders.keeptrack")) {
-		        glblSenderKeepTrack = (int) cnfparamvals[i].val.d.n;
+	        glblSenderKeepTrack = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "inputs.timeout.shutdown")) {
-		        glblInputTimeoutShutdown = (int) cnfparamvals[i].val.d.n;
+	        glblInputTimeoutShutdown = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "privdrop.group.keepsupplemental")) {
-		        loadConf->globals.gidDropPrivKeepSupplemental = (int) cnfparamvals[i].val.d.n;
+	        loadConf->globals.gidDropPrivKeepSupplemental = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "net.acladdhostnameonfail")) {
-		        *(net.pACLAddHostnameOnFail) = (int) cnfparamvals[i].val.d.n;
+	        *(net.pACLAddHostnameOnFail) = (int) cnfparamvals[i].val.d.n;
 		} else if(!strcmp(paramblk.descr[i].name, "net.aclresolvehostname")) {
-		        *(net.pACLDontResolve) = !((int) cnfparamvals[i].val.d.n);
+	        *(net.pACLDontResolve) = !((int) cnfparamvals[i].val.d.n);
 		} else if(!strcmp(paramblk.descr[i].name, "net.enabledns")) {
-		        setDisableDNS(!((int) cnfparamvals[i].val.d.n));
+	        setDisableDNS(!((int) cnfparamvals[i].val.d.n));
```

Comment:
any specific reason you removed the indention? With it, the code is far more readable.

Reply:
Actually I corrected the indention... After careful inspection, though, these lines had some spaces instead of tabs. GitHub was displaying a different indention than my editor because of the tab size (8 in GitHub instead of 4 in my editor). I have replaced the spaces with tabs and updated the PR.

BTW, do you have a specific convention for the tab size in which the code is supposed to be viewed? Sometimes there can be differences, as in this case. (Note: people advocate for avoiding the use of tabs in open source projects to avoid this kind of issues, although I personally have no problem with them...)